include/usgscsm/UsgsAstroLsSensorModel.h

```diff
@@ -125,6 +125,7 @@ class UsgsAstroLsSensorModel : public csm::RasterGM,
   double m_halfTime;
   std::vector<double> m_covariance;
   int m_imageFlipFlag;
+  double m_stepSizeFactor;
```

Comment:
This needs to be serialized into the model state like the other computed members. It can be an optional value on the read back in to preserve backwards compatibility.